co-log-core/src/Colog/Core/Action.hs

```diff
@@ -154,6 +165,14 @@ cmap :: (a -> b) -> LogAction m b -> LogAction m a
 cmap f (LogAction action) = LogAction (action . f)
 {-# INLINE cmap #-}
 
+{- | This combinator is @>$@ from contravariant functor. Replaces all locations
+in the output with the same value. The default definition is
+@contramap . const@, so this is a more efficient version.
+-}
+infixl 4 >$
+(>$) :: b -> LogAction m b -> LogAction m a
+(>$) f (LogAction action) = LogAction (const (action f))
```

Comment:
I guess you can rename `f` to `b`. And I think that in order to make this function more efficient than `contramap . const`, it's better to write lambda instead of using `const` function. Though, the performance difference might not be even noticeable. Though, still, it's good to do it explicitly.